src/main/java/com/puppycrawl/tools/checkstyle/utils/CodePointUtil.java

```diff
@@ -42,4 +42,25 @@ public static boolean isBlank(int... codePoints) {
                 .allMatch(Character::isWhitespace);
     }
 
+    /**
+     * Returns whether the given code point array contains
+     * only whitespace up to the specified index.
+     *
+     * @param codePoints
+     *            array of Unicode code point of string to check
+     * @param index
```

Comment:
perhaps add `index to check up to (exclusive)`, off by one errors always confuse me.

Reply:
done

Reply:
Resolved.